packages/fxa-auth-server/lib/googlemapsservices.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  AddressComponent,
+  AddressType,
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { Logger } from 'mozlog';
+import { Container } from 'typedi';
+
+import { ConfigType } from '../config';
+
+export type ZipLocation = {
+  city: string;
+  country: string;
+  countryCode: string;
+  state: string;
+  stateCode: string;
+};
+
+export class GoogleMapsService {
+  private log: Logger;
```

Comment:
There's an `AuthLogger` type that we setup that has some additional setup on it, see:
https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/payments/capability.ts#L45

Note that the `AuthLogger` type can be fetched.